frontend/test/pytest/test_mid_circuit_measurement.py

```diff
@@ -569,6 +569,9 @@ def outer():
         assert result.shape == (shots,)
         assert jnp.allclose(result, 1.0)
 
+    @pytest.mark.skip(
```

Comment:
Let's mark xfail for now, at least if something crashes we'll know 😅 
```suggestion
    @pytest.mark.xfail(
```

Reply:
done